### PR TITLE
Exception handling when rqlite node is unavailable

### LINKED ISSUE #12: Handling case where rqlite node is not available
In my previous PR, I preserved original behavior for when we can't connect to a node: rqlite-java prints a message that it couldn't connect to a node, then returns null.  I'd like to change it to print nothing and throw an exception that the client code can handle.  Is that okay?

### PR DESCRIPTION
**Pull request by hprovenza**
Fixes #12

### CODE REVIEW

File: src/main/java/com/rqlite/Rqlite.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import com.rqlite.dto.ExecuteResults;
 import com.rqlite.dto.Pong;
 import com.rqlite.dto.QueryResults;
+import com.rqlite.impl.NodeUnavailableException;
```
Comment (otoolep) [inline src/main/java/com/rqlite/Rqlite.java]: `impl` in the package name implies that details of those package should not be exposed. Shouldn't this exception be defined elsewhere?

File: src/main/java/com/rqlite/impl/NodeUnavailableException.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package com.rqlite.impl;
+
+/**
+ * This exception is thrown when rqlite-java cannot connect to a rqlite node.
+ **/
+public class NodeUnavailableException extends Exception {
```
Comment (otoolep) [inline src/main/java/com/rqlite/impl/NodeUnavailableException.java]: See comment above. I think this should be defined outside of the implementation, in the same place the interface is defined.

**Comment by otoolep** [comment]
Let me know when you're ready for another review.

**Comment by hprovenza** [comment]
Thanks - I think I just have to clean up the commit history at this point.

**Comment by hprovenza** [comment]
Go ahead!

**Review by otoolep — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**